### ISSUE TITLE
No X11 DISPLAY variable was set

### ISSUE THREAD

**Issue by ankurldots** [apache/jmeter#5751]
### Expected behavior

I downloaded the latest binary and tried to run Jmeter. However I got the error as described in actual behaviour.

Expected behaviour was - Jmeter GUI should open. 

### Actual behavior

ankur@ankur:~/software/apache-jmeter-5 (1).5/bin$ ./jmeter
================================================================================
Don't use GUI mode for load testing !, only for Test creation and Test debugging.
For load testing, use CLI Mode (was NON GUI):
   jmeter -n -t [jmx file] -l [results file] -e -o [Path to web report folder]
& increase Java Heap to meet your test requirements:
   Modify current env variable HEAP="-Xms1g -Xmx1g -XX:MaxMetaspaceSize=256m" in the jmeter batch file
Check : https://jmeter.apache.org/usermanual/best-practices.html
================================================================================
An error occurred: 
**No X11 DISPLAY variable was set,
or no headful library support was found,
but this program performed an operation which requires it,**

### Steps to reproduce the problem

system information

![image](https://user-images.githubusercontent.com/69628422/209275279-d33894d1-03ec-46cd-ac46-a591551b8060.png)

My display variable is also set - 
ankur@ankur:~/software/apache-jmeter-5 (1).5/bin$ echo $DISPLAY 
:1

I am using dual display, is that the issue?

### JMeter Version

5.5

### Java Version

openjdk version "19.0.1" 2022-10-18 OpenJDK Runtime Environment (build 19.0.1+10-Ubuntu-1ubuntu122.04) OpenJDK 64-Bit Server VM (build 19.0.1+10-Ubuntu-1ubuntu122.04, mixed mode, sharing)

### OS Version

Ubuntu 22.04.1 LTS

**Comment by FSchumacher** [comment]
I never tried to use a GUI from within wsl, but the error message says: 

> No X11 DISPLAY variable was set, or no headful library support was found,

So, if you have set the DISPLAY variable, it could be, that you have not installed the needed X11 libraries or a JRE with the needed X11 parts. In Ubuntu there are JRE/JDK packages, that exclude those parts. Which packages do you use (have installed)?

**Comment by ankurldots** [comment]
Its Jmeter, so i dont know what packages? can i run some command and give you more information?

**Comment by FSchumacher** [comment]
No, it is not JMeter. It is your Java installation. Look for openjdk packages.
